newIDE/app/src/AssetStore/BehaviorStore/index.js

```diff
@@ -0,0 +1,270 @@
+        project.hasEventsFunctionsExtensionNamed(
+          behaviorShortHeader.extensionName
+        );
+      if (isExtensionAlreadyInstalled) {
+        const shouldUpdateExtension = await showExtensionUpdateConfirmation();
+        if (!shouldUpdateExtension) {
+          return;
+        }
+      }
+
+      if (behaviorShortHeader.url) {
+        sendExtensionAddedToProject(behaviorShortHeader.name);
+        const wasInstalled = await onInstall(behaviorShortHeader);
+        // An errorBox is already displayed by `installExtension`.
+        if (wasInstalled) {
+          onChoose(behaviorShortHeader.type);
+        }
+      } else {
+        onChoose(behaviorShortHeader.type);
+      }
+    },
+    [project, onChoose, showExtensionUpdateConfirmation, onInstall]
+  );
+
+  return (
+    <React.Fragment>
+      <ColumnStackLayout expand noMargin useFullHeight>
+        <ColumnStackLayout noMargin>
+          <ResponsiveLineStackLayout noMargin>
+            <SearchBarSelectField
+              value={chosenCategory}
+              onChange={(e, i, value: string) => {
+                setChosenCategory(value);
+              }}
+            >
+              <SelectOption value="" label={t`All categories`} />
+              {allCategories.map(category => (
+                <SelectOption
+                  key={category}
+                  value={category}
+                  label={category}
+                />
+              ))}
+            </SearchBarSelectField>
+            <Line expand noMargin>
+              <Column expand noMargin>
+                <SearchBar
+                  id="extension-search-bar"
+                  value={searchText}
+                  onChange={setSearchText}
+                  onRequestSearch={() => {}}
+                  tagsHandler={tagsHandler}
+                  tags={filters && filters.allTags}
+                  placeholder={t`Search extensions`}
+                  autoFocus="desktop"
+                />
+              </Column>
+              <ElementWithMenu
+                key="menu"
+                element={
+                  <IconButton size="small">
+                    <ThreeDotsMenu />
+                  </IconButton>
+                }
+                buildMenuTemplate={(i18n: I18nType) => [
+                  {
+                    label: preferences.values.showCommunityExtensions
+                      ? i18n._(
+                          t`Hide community extensions (not officially reviewed)`
+                        )
+                      : i18n._(
+                          t`Show community extensions (not officially reviewed)`
+                        ),
+                    click: () => {
+                      preferences.setShowCommunityExtensions(
+                        !preferences.values.showCommunityExtensions
+                      );
+                    },
+                  },
+                  {
+                    label: showDeprecated
```

Comment:
Do we still have a concept of deprecated behaviors? If so which ones?

Reply:
It's only for Physics 1. There is some community extensions that have "deprecated" in their names but they won't be considered deprecated (which is not an issue because they already are in the community list which is hidden by default).